packages/graphql/src/translate/utils/math.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import mapToDbProperty from "../../utils/map-to-db-property";
+import { GraphElement } from "../../classes";
+
+/** Maps Neo4jGraphQL Math operator to Cypher symbol */
+const CypherOperatorMap = new Map<string, string>([
+    ["_ADD", "+"],
+    ["_SUBTRACT", "-"],
+    ["_MULTIPLY", "*"],
+    ["_DIVIDE", "/"],
+    ["_INCREMENT", "+"],
+    ["_DECREMENT", "-"]
+]);
+
+function mathOperatorToSymbol(mathOperator: string): string {
+    if (CypherOperatorMap.has(mathOperator)) {
+        return CypherOperatorMap.get(mathOperator) as string;
+    }
+    throw new Error(`${mathOperator} is not a valid math operator`);
+}
+
+export const MATH_FIELD_REGX = /(?<propertyName>\w*)(?<operatorName>_INCREMENT|_DECREMENT|_ADD|_SUBTRACT|_DIVIDE|_MULTIPLY)\b/;
+
+interface MathDescriptor {
+    dbName: string;
+    graphQLType: string;
+    fieldName: string;
+    operationName: string;
+    operationSymbol: string;
+    value: number;
+}
+
+interface MathMatch {
+    isMatched: boolean;
+    operatorName: string;
+    propertyName: string;
+}
+// Returns True in case of a valid match and the potential match. 
+export function matchMathField(graphQLFieldName: string): MathMatch {
+    const mathFieldMatch = graphQLFieldName.match(MATH_FIELD_REGX);
+    if (mathFieldMatch && mathFieldMatch.groups) {
+        const { operatorName, propertyName } = mathFieldMatch.groups;
+        const isMatched = Boolean(mathFieldMatch && mathFieldMatch.length > 2 && operatorName && propertyName);
```

Comment:
Maybe `hasMatched` or `isMathField` are better names here?

Reply:
Both better than  `isMatched` for sure :)